modules/configuring-scale-bounds-knative.adoc

```diff
@@ -12,15 +12,16 @@ minScale:: If the `minScale` annotation is not set, Pods will scale to zero (or
 
 maxScale:: If the `maxScale` annotation is not set, there will be no upper limit for the number of Pods created.
 
-`minScale` and `maxScale` can be configured as follows in the revision template:
+`minScale` and `maxScale` can be configured as follows in the revision template (`spec.template.metadata.annotations`):
```

Comment:
I know this is a nitpick, but I don't think that this addition in parentheses adds any value. We're showing in the example immediately following where these fields are configured at. Unless @abrennan89 thinks that they are helpful, I'd suggest to remove the change to this line. @abrennan89 wdyt?

Reply:
Yeah @bergerhoffer I'd say that's fair since it's added to the actual example now, probably doesn't need both. @nak3 can you update just to remove the ( ) part?

Reply:
Sure, no problem! Updated.